user_b/analytics_logging.py

```diff
@@ -0,0 +1,51 @@
+from enum import Enum
+import uuid
+from app import db
+from app.errors.errors import DatabaseError
+from app.models import UserBAnalyticsData
+from datetime import datetime, timezone
+from enum import Enum
+
+
+class eventType(Enum):
+    """
+    Event types to be used for analytics logging.

```

Comment:
I think it’s best that analytic event types and names be decided by an analytics team member or in collaboration with analytics team member to minimize need for code changes.